src/screens/home/HomeScreen.tsx

```diff
@@ -89,33 +89,35 @@ const Content = ({setBackgroundColor, isBottomSheetExpanded}: ContentProps) => {
   switch (systemStatus) {
     case SystemStatus.Undefined:
       return null;
-    case SystemStatus.BluetoothOff:
-      return <BluetoothDisabledView />;
     case SystemStatus.Disabled:
```

Comment:
Would these case also create the same issue of hiding code uploading instructions to the user? Or do these system status also prevent code uploading?

Reply:
This case is only for when the user is in the "monitoring" state, and would not apply when the user needs to upload codes.